react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,334 @@
+        enabled: true,
+        html: <span className = 'badge-round'>
+            <span id = 'numberOfParticipants' />
+        </span>,
+        id: 'toolbar_contact_list',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent(
+                'toolbar.contacts.toggled');
+            APP.UI.emitEvent(UIEvents.TOGGLE_CONTACT_LIST);
+        },
+        sideContainerId: 'contacts_container',
+        tooltipKey: 'bottomtoolbar.contactlist'
+    },
+    profile: {
+        classNames: [ 'button' ],
+        enabled: true,
+        html: <img
+            id = 'avatar'
+            src = 'images/avatar2.png' />,
+        id: 'toolbar_button_profile',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.profile.toggled');
+            APP.UI.emitEvent(UIEvents.TOGGLE_PROFILE);
+        },
+        sideContainerId: 'profile_container',
+        tooltipKey: 'profile.setDisplayNameLabel'
+    },
+    etherpad: {
+        classNames: [ 'button', 'icon-share-doc' ],
+        enabled: true,
+        hidden: true,
+        id: 'toolbar_button_etherpad',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.etherpad.clicked');
+            APP.UI.emitEvent(UIEvents.ETHERPAD_CLICKED);
+        },
+        tooltipKey: 'toolbar.etherpad'
+    },
+    fullscreen: {
+        classNames: [ 'button', 'icon-full-screen' ],
+        content: 'Enter / Exit Full Screen',
+        enabled: true,
+        i18n: 'toolbar.fullscreen',
+        id: 'toolbar_button_fullScreen',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.fullscreen.enabled');
+
+            APP.UI.emitEvent(UIEvents.TOGGLE_FULLSCREEN);
+        },
+        shortcut: 'S',
+        shortcutAttr: 'toggleFullscreenPopover',
+        shortcutDescription: 'keyboardShortcuts.fullScreen',
+        shortcutFunc() {
+            JitsiMeetJS.analytics.sendEvent('shortcut.fullscreen.toggled');
+            APP.UI.toggleFullScreen();
+        },
+        tooltipKey: 'toolbar.fullscreen'
+    },
+    settings: {
+        classNames: [ 'button', 'icon-settings' ],
+        enabled: true,
+        id: 'toolbar_button_settings',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.settings.toggled');
+            APP.UI.emitEvent(UIEvents.TOGGLE_SETTINGS);
+        },
+        sideContainerId: 'settings_container',
+        tooltipKey: 'toolbar.Settings'
+    },
+    hangup: {
+        classNames: [ 'button', 'icon-hangup', 'button_hangup' ],
+        content: 'Hang Up',
+        enabled: true,
+        i18n: 'toolbar.hangup',
+        id: 'toolbar_button_hangup',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.hangup');
+            APP.UI.emitEvent(UIEvents.HANGUP);
+        },
+        tooltipKey: 'toolbar.hangup'
+    },
+    raisehand: {
+        classNames: [ 'button', 'icon-raised-hand' ],
+        content: 'Raise Hand',
+        enabled: true,
+        i18n: 'toolbar.raiseHand',
```

Comment:
I don't think that `i18n` is effectively used anywhere anymore. I changed an use of `i18n` to `key` in my commit  in order to fix a runtime error. Looking back now, I think I may have broken `i18n` but I see that not every button has `i18n` and your use of it which raised the runtime error was because you relied that `i18n` was always defined. Can you please figure out what's going on with `i18n` and make sure that it's actually eventually used?